### PR TITLE
cleanup: fix spelling mistakes in CHANGELOG-1.20.md

### PR DESCRIPTION
**Pull request by wangyx1992**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
This pr fixes spelling mistakes in CHANGELOG-1.20.md.

#### Which issue(s) this PR fixes:
Fixes #

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

**Review by soniasingla — APPROVED** [review]
/lgtm
/approve

**Comment by wilsonehusin** [comment]
hey @wangyx1992, thanks for submitting this PR. in general, this falls under [trivial edits] which are not encouraged to be merged. I would have an easier time justifying the approval if there are more fixes provided in this PR. would you like to give the 1.20 changelog another check on other spelling issues?

**Comment by wangyx1992** [comment]
> hey @wangyx1992, thanks for submitting this PR. in general, this falls under [trivial edits] which are not encouraged to be merged. I would have an easier time justifying the approval if there are more fixes provided in this PR. would you like to give the 1.20 changelog another check on other spelling issues?

Hi, @wilsonehusin, I have checked the 1.20 changelog and modify found spelling issuses already, waiting for your review with pleasure.

File: CHANGELOG/CHANGELOG-1.20.md
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:429]: this should be expanded, i.e.

```suggestion
  - Skip non-kubemark nodes while computing node information for node groups.
```

File: CHANGELOG/CHANGELOG-1.20.md
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:1145]: hmm I think this should be `CronJob`

File: CHANGELOG/CHANGELOG-1.20.md
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:1298]: I think `Dockerfiles` is right here, plural seem intentional

File: CHANGELOG/CHANGELOG-1.20.md
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:1442]: same as previous, `CronJob`

File: CHANGELOG/CHANGELOG-1.20.md
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:1813]: same as above, i.e. `CronJob`

File: CHANGELOG/CHANGELOG-1.20.md
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:1897]: same as above, i.e. `CronJob`

File: CHANGELOG/CHANGELOG-1.20.md
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:2668]: same as above, i.e. `Dockerfiles`

**Review by wilsonehusin — CHANGES_REQUESTED** [review]
thanks for taking the time and reviewing the whole document! small changes and then we should be good to go 😄

File: CHANGELOG/CHANGELOG-1.20.md
Comment (wangyx1992) [reply CHANGELOG/CHANGELOG-1.20.md]: Yeah, corresponding to the before `image names`, we should `Dockerfiles` use here.

File: CHANGELOG/CHANGELOG-1.20.md
Comment (wangyx1992) [reply CHANGELOG/CHANGELOG-1.20.md]: Hi, @wilsonehusin, all abolve have been updated already, looking for your review with pleasure.

**Review by wilsonehusin — APPROVED** [review]
awesome, thanks!

/approve

**Pull request merged**
